Formula/rxvt-unicode.rb

```diff
@@ -39,6 +46,17 @@ class RxvtUnicode < Formula
   end
 
   def install
+    ENV.cxx11
+    resource("libptytty").stage do
+      system "cmake", "-S", ".", "-B", "build", *std_cmake_args, "-DBUILD_SHARED_LIBS=ON"
+      system "cmake", "--build", "build"
+      system "cmake", "--install", "build"
+    end
+    ENV.prepend_path "PKG_CONFIG_PATH", lib/"pkgconfig"
+    # Add an RPATH so rxvt-unicode can find libptytty on macOS.
+    # The RPATH is already added automatically the compiler shim in Linux.
+    ENV.append "LDFLAGS", "-Wl,-rpath,#{lib}" if OS.mac?
```

Comment:
```suggestion
    ENV.append "LDFLAGS", "-Wl,-rpath,#{rpath}" if OS.mac?
```
?

Reply:
Since we should really be installing this in `libexec` instead of the main prefix, this will become:
```ruby
ENV.append "LDFLAGS", "-Wl,-rpath,#{libexec}/lib"
```
and will be needed on both OSes.

Reply:
Try
```ruby
ENV.append "LDFLAGS", "-Wl,-rpath,#{rpath(target: libexec/"lib")"
```